src/ocean/util/serialize/contiguous/package_.d

```diff
@@ -696,3 +697,46 @@ unittest
     auto d = Deserializer.deserialize!(S2)(buffer);
     test(deepEquals(*d.ptr, s));
 }
+
+/******************************************************************************
+
+    Const arrays of arrays
+
+******************************************************************************/
+
+struct ConstS
+{
+    Const!(int) n = 3;
+    Const!(char[][]) a = ["Hello", "World"];
+    Const!(char[])[2][3] b = [
+        ["Die", "Katze"], ["tritt", "die"], ["Treppe", "krumm."]
+    ];
+}
+
+unittest
+{
+    void[] buffer;
+
+    ConstS s;
+
+    Serializer.serialize(s, buffer);
+    auto cont_S = Deserializer.deserialize!(ConstS)(buffer);
```

Comment:
Please also check with @gavin-norman-sociomantic - we have just had a slack discussion about this and he went to try another approach in dhtproto. It looks like things got a bit of out of control because I expected deserializer to refuse const types and it silently ingored them instead :( Going to see if that alternative approach proves feasible before making decision about serializer.